README.md

```diff
@@ -181,6 +181,10 @@ For Windows, [we release images](#windows-images) only for amd64 CPU architectur
 - Windows Server 2019
 - Windows Server 2022
 
+#### debug images
+
+The Fluent Bit Debug Image is a containerized environment designed for testing and debugging Fluent Bit configurations, and it also has the capability to transfer core dump files to an S3 bucket for further analysis and troubleshooting. This feature can help streamline the debugging process and facilitate quicker resolution of issues.
```

Comment:
give the tag format for the debug images

Reply:
Also I'd prefer to rephrase this as:

> Deploying AWS for Fluent debug images can help the AWS team troubleshoot an issue. If you experience a bug, especially a crash/SIGSEGV issue (link to debugging guide SIGSEGV section), then please consider deploying the debug version of the image. After a crash, the debug image can print out a stacktrace and upload a core dump to S3. See our debugging guide {link} for more info on using debug images.

Reply:
We possibly should explicitly say not to deploy these to prod... I also still want to run the load tests on them though actually and call that out... I really think it should be safe to deploy these in prod...